### PR TITLE
feat(sbb-lead-container): add support for sbb-alert-group

### LINKED ISSUE #2714: Alert Box in Lead Container
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/sbb-design-systems/lyne-components/blob/main/docs/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/4) for an issue that matches the one I want to file, without success.

### Request type

Request for enhancement of a component

### It affects the following packages

Components

### Functionality

Lead Container: please include the Alert Box in the Lead Container above the Breadcrumb
Thanks

### Link to design proposal file

_No response_

### PR DESCRIPTION
**Pull request by mcilurzo**
added css class
updated story and documentation

Closes #2714

### CODE REVIEW

File: src/elements/lead-container/lead-container.visual.spec.ts [hunk 1]
```diff
@@ -31,6 +32,15 @@ describe(`sbb-lead-container`, () => {
         }
       </style>
       ${image}
+      <sbb-alert-group class="sbb-lead-container-spacing">
```
Comment (jeripeierSBB) [inline src/elements/lead-container/lead-container.visual.spec.ts:35]: I remember that Florian told me, the alert group should be after the breadcrumb. Did you discuss this?

Comment (mcilurzo) [reply src/elements/lead-container/lead-container.visual.spec.ts:35]: I checked it with florian. it is OK as it is.

**Review by jeripeierSBB — APPROVED** [review]
Great! :-)

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
Nice work! 😃

**Pull request merged**